doc/changelog.md

```diff
@@ -61,15 +61,14 @@
   [(#579)](https://github.com/PennyLaneAI/catalyst/pull/579)
 
   ```py
-  from catalyst.oqc import OQCDevice
 
   import os
 
   os.environ["OQC_EMAIL"] = "your_email"
   os.environ["OQC_PASSWORD"] = "your_password"
   os.environ["OQC_URL"] = "oqc_url"
 
-  device = OQCDevice(backend="lucy", shots=2012, wires=2)
+  device = qml.device("oqc.remote", backend="lucy", shots=2012, wires=2)
```

Comment:
Only suggestion @rmoyard is to update this to `oqc.cloud` to match the name of their cloud service https://oqc.tech/access/oqc-cloud/

Reply:
Ok for sure :+1: